cmd/kubeadm/app/phases/upgrade/preflight.go

```diff
@@ -52,10 +51,7 @@ func (c CoreDNSCheck) Check() (warnings, errors []error) {
 }
 
 // RunCoreDNSMigrationCheck initializes checks related to CoreDNS migration.
-func RunCoreDNSMigrationCheck(client clientset.Interface, ignorePreflightErrors sets.String, dnsType kubeadmapi.DNSAddOnType) error {
-	if dnsType != kubeadmapi.CoreDNS {
-		return nil
-	}
```

Comment:
i don't think we need these migration checks for kube-dns.

Reply:
Probably yes, but it's too late in the cycle to do anything about them.